client/src/PlayerLanding/PlayerInGame.tsx

```diff
@@ -82,6 +92,17 @@ export function PlayerInGame() {
         </Tabs.Trigger>
       </Tabs.List>
 
+      {selectedTab === "night-order" && (
+        <Button
+          className="sticky mx-2 my-1"
+          size="1"
+          variant={isFirstNightSort ? "solid" : "outline"}
+          onClick={() => setIsFirstNightSort((prev) => !prev)}
+        >
+          {isFirstNightSort ? "First Night" : "Other Night"}
+        </Button>
+      )}
```

Comment:
This is dumb, but "sticky" doesn't do anything within a Tabs.Content, and I believe this indicator should always be visible to the player.